business/sys/database/database.go

```diff
@@ -78,6 +78,15 @@ func Open(cfg Config) (*sqlx.DB, error) {
 // StatusCheck returns nil if it can successfully talk to the database. It
 // returns a non-nil error otherwise.
 func StatusCheck(ctx context.Context, db *sqlx.DB) error {
+	// If there is no timeout in parent context
```

Comment:
Line space above a comment and proper sentence structure. This comment could probably be all on one line.